sdk/formrecognizer/azure-ai-formrecognizer/src/samples/java/com/azure/ai/formrecognizer/AdvancedDiffLabeledUnlabeledData.java

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.formrecognizer;
+
+import com.azure.ai.formrecognizer.models.FormContentType;
+import com.azure.ai.formrecognizer.models.RecognizedForm;
+import com.azure.core.credential.AzureKeyCredential;
+import com.azure.core.util.IterableStream;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
+/**
+ * This sample demonstrates the differences in output that arise when recognize_custom_forms
+ * is called with custom models trained with labeled and unlabeled data.
+ */
+public class AdvancedDiffLabeledUnlabeledData {
+
+    /**
+     * Main method to invoke this demo.
+     *
+     * @param args Unused arguments to the program.
+     *
+     * @throws IOException Exception thrown when there is an error in reading all the bytes from the File.
+     */
+    public static void main(String[] args) throws IOException {
+        // Instantiate a client that will be used to call the service.
+
+        FormRecognizerClient client = new FormRecognizerClientBuilder()
+            .apiKey(new AzureKeyCredential("{api_Key}"))
+            .endpoint("https://{endpoint}.cognitiveservices.azure.com/")
+            .buildClient();
+
+        File analyzeFile = new File("../../test/resources/sample-files/Invoice_1.pdf");
```

Comment:
I think it'd be best to have the forms also in the samples folder, doesn't look great going into the test folder